packages/test/test-end-to-end-tests/.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"label": "Build Current Tests",
+			"type": "process",
+			"command": "node",
+			"args": [
+				"${workspaceRoot:FluidFramework}/node_modules/@fluidframework/build-tools/dist/fluidBuild/fluidBuild.js",
```

Comment:
Would this read cleaner with the first arg being `fluid-build` and adding the bit below? The current form assumes the root has its deps installed, and in that case calling `fluid-build` directly should work, no?

```
"options": {
  "cwd": "${workspaceRoot:FluidFramework}"
}
```

Reply:
And if that works, I guess remove the --root flag.

Reply:
can't be the first arg, it needs to be the command, but I believe we could do this if we wanted:

![image](https://github.com/user-attachments/assets/886ea4ab-a72b-4991-b3df-3eccf94959cc)

I copied this task from the root tasks.json (annoyingly we don't inherit the definitions there, just the launch targets AFAICT), so if we want to do this we should do it there as well.

Reply:
Oh right, replace the node invocation entirely, yep.